application/src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -297,6 +311,20 @@ type BookingTime {
   time: String
 }
 
+"Real-time estimates for arrival and departure times for a stop location."
+type CallRealTime {
+  "Real-time estimates for the arrival."
+  arrival: EstimatedTime
+  "Real-time estimates for the departure."
+  departure: EstimatedTime
+}
+
+"What is scheduled for a trip on a service date for a stop location."
+type CallSchedule {
```

Comment:
What is the reason for wrapping this in another type?

Reply:
We can later include information if there is a planned cancellation, for example.